src/noisepy/seis/io/channelcatalog.py

```diff
@@ -178,4 +182,127 @@ def get_inventory(self, timespan: DateTimeRange, station: Station) -> obspy.Inve
         return obspy.Inventory(nets)
 
 
+def sta_info_from_inv(inv: obspy.core.inventory.inventory.Inventory):
+    """
+    this function outputs station info from the obspy inventory object
+    (used in S0B)
+    PARAMETERS:
+    ----------------------
+    inv: obspy inventory object
+    RETURNS:
+    ----------------------
+    sta: station name
+    net: netowrk name
+    lon: longitude of the station
+    lat: latitude of the station
+    elv: elevation of the station
+    location: location code of the station
+    """
+    # load from station inventory
+    sta = inv[0][0].code
+    net = inv[0].code
+    lon = inv[0][0].longitude
+    lat = inv[0][0].latitude
+    if inv[0][0].elevation:
+        elv = inv[0][0].elevation
+    else:
+        elv = 0.0
+
+    if inv[0][0][0].location_code:
+        location = inv[0][0][0].location_code
+    else:
+        location = "00"
+
+    return sta, net, lon, lat, elv, location
+
+
+def stats2inv_mseed(stats, locs: pd.DataFrame) -> Inventory:
+    inv = Inventory(networks=[], source="homegrown")
+    ista = locs[locs["station"] == stats.station].index.values.astype("int64")[0]
+
+    net = Network(
+        # This is the network code according to the SEED standard.
+        code=locs.iloc[ista]["network"],
+        stations=[],
+        description="created from SAC and resp files",
+        start_date=stats.starttime,
+    )
+
+    sta = st(
```

Comment:
Given that the type is only used in one place I think it's clearer to just use the full name to avoid the collision, same below
```suggestion
    sta = obspy.core.inventory.Station(
```